include/sound/sof/stream.h

```diff
@@ -86,9 +86,11 @@ struct sof_ipc_stream_params {
 	uint32_t host_period_bytes;
 	uint16_t no_stream_position; /**< 1 means don't send stream position */
 	uint8_t cont_update_posn; /**< 1 means continuous update stream position */
-
-	uint8_t reserved[5];
+	uint8_t reserved0;
+	int16_t ext_data_length; /**< 0, means no extended data */
```

Comment:
It might have been already answered, but why we need the `reserved0`? To bring the ext_data_length to 16bit alignment?

Reply:
for that one could just place `ext_data_length` after the 3 reserved bytes.

Reply:
Unfortunately the change was already merged in the FW side so we cannot change it here. Sorry for late answer I just came back from vacation :)